.ci/.eslintplugin.js

```diff
@@ -0,0 +1,167 @@
+const fs = require('fs')
```

Comment:
Add semicolons throughout

Reply:
Done, and a check added on the linter itself for this.